test/integration/targets/bigip_policy_rule/tasks/issue-01678.yaml

```diff
@@ -0,0 +1,77 @@
+---
+
+- name: Issue 01678 - Create published policy
+  bigip_policy:
+      name: issue-01678
+
+- name: Issue 01678 - Create published rule with set_variable actions
+  bigip_policy_rule:
+      policy: issue-01678
+      name: rule1
+      actions:
+          - type: set_variable
+            variable_name: user-agent
+            expression: tcl:[HTTP::header User-Agent]
```

Comment:
2 spaces not 4

Reply:
I have fixed indentation - this comment is addressed now